InnerEye/ML/utils/split_dataset.py

```diff
@@ -183,48 +206,87 @@ def get_subject_ranges_for_splits(population: Sequence[str],
         }
         return result
 
+    @staticmethod
+    def _from_split_keys(df: pd.DataFrame,
+                         train_keys: Sequence[str],
+                         test_keys: Sequence[str],
+                         val_keys: Sequence[str],
+                         *,  # make column names keyword-only arguments to avoid mistakes when providing both
+                         key_column: str,
+                         subject_column: str,
+                         group_column: Optional[str]) -> DatasetSplits:
+        """
+        Takes a slice of values from each data split train/test/val for the provided keys.
```

Comment:
What's a slice of values?

Reply:
I just adapted this docstring from `from_subject_ids()`.